torch/csrc/jit/passes/quantization.cpp

```diff
@@ -144,21 +144,21 @@ Node* createIntReprNode(Value* v, Node* n) {
 }
 
 // Insert Quant-Dequant node pattern for quantizable node outputs
-void addQuantDeQuantNodes(
+Node* addQuantDeQuantNodesFor(
     Value* v,
+    Node* insert_point,
     const std::tuple<std::string, float, int>& qparam,
     at::ScalarType t = at::ScalarType::Undefined) {
   TORCH_INTERNAL_ASSERT(v != nullptr);
-  Node* n = v->node();
-  Node* quant = createQuantNode(v, n);
-  Node* intrepr = createIntReprNode(v, n);
-  Node* dequant = createDeQuantNode(v, n);
+  WithInsertPoint ins(insert_point);
+  Node* quant = createQuantNode(v, insert_point);
+  Node* intrepr = createIntReprNode(v, insert_point);
+  Node* dequant = createDeQuantNode(v, insert_point);
 
   // Add quant-intrepr-dequant nodes and replace for all uses of Value
-  quant->insertAfter(n);
+  quant->insertAfter(insert_point);
```

Comment:
Nit: since you've already set up the insert point, you could use `insert` instead of `insertAfter`. That would actually help to make sure we're inserting everything in the same place.

Reply:
btw are u referring to below api.  I am trying to look for usage for below api in code.  will setting the insert_point not guarantee all nodes inserted at same point ? 
TORCH_API Value* insert(
      Symbol opname,
      at::ArrayRef<NamedValue> args,
      at::ArrayRef<NamedValue> kwargs = {},
      const c10::optional<SourceRange>& range = {});

Reply:
i have retained same api for same reason. insert requires passing explicit opname which we want to avoid as quant node may refer to quantize_linear or quantize_linear_affine in coming future

Reply:
There are still ways to pull it off. One potential way would be to figure out the symbol basing on the qconfig/qparams and then passing it to insert - I don't see why it would be a problem. What concerns me is that we are currently using two ways for nodes insertion - using only one would make the code more robust for potential future changes.

Reply:
sorry i did not follow what are the two ways :) for node insertion. I also wanted to match the mechanism we use in addObserver function that u added. There we use insertAfter instead of insert. I think if we are to use insert, it should be same everywhere - agree

Reply:
since it's only a nitpick thing according to @ZolotukhinM , let's just remove `WithInsertionPoint` and keep `insertAfter` for now.